docs/platform-integration/essentials/preferences.md

```diff
@@ -1,107 +1,141 @@
+[NSUserDefaults](../ios/app-fundamentals/user-defaults.md) is used to store values on iOS devices. If no `sharedName` is specified, the `StandardUserDefaults` are used. Otherwise, the name is used to create a new `NSUserDefaults` with the specified name used for the `NSUserDefaultsType.SuiteName`.
 
 # [Windows](#tab/windows)
 
-[ApplicationDataContainer](/uwp/api/windows.storage.applicationdatacontainer) is used to store the values on the device. If no `sharedName` is specified the `LocalSettings` are used, otherwise the name is used to create a new container inside of `LocalSettings`.
+[ApplicationDataContainer](/uwp/api/windows.storage.applicationdatacontainer) is used to store the values on the device. If no `sharedName` is specified, the `LocalSettings` are used. Otherwise the name is used to create a new container inside of `LocalSettings`.
 
-`LocalSettings` also has the following restriction that the name of each setting can be 255 characters in length at most. Each setting can be up to 8K bytes in size and each composite setting can be up to 64K bytes in size.
+<!-- TODO: This makes no sense to me. the way the word setting was used here didn't make sense. I edited this to try and make sense of it but I suspect I'm wrong! -->
+`LocalSettings` restricts the preference key names to 255 characters or less. Each preference value can be up to 8K bytes in size, and each composite setting can be up to 64K bytes in size.
 
---------------
+-----
+<!-- markdownlint-enable MD025 -->
 
 ## Persistence
 
-Uninstalling the application will cause all _Preferences_ to be removed, with the exception being apps that target and run on Android 6.0 (API level 23) or later that use [__Auto Backup__](https://developer.android.com/guide/topics/data/autobackup). This feature is on by default and preserves app data including __Shared Preferences__, which is what the **Preferences** API utilizes. You can disable this by following Google's [documentation](https://developer.android.com/guide/topics/data/autobackup).
+Uninstalling the application causes all _preferences_ to be removed, except when the app runs on Android 6.0 (API level 23) or later, while using the [**Auto Backup**`](https://developer.android.com/guide/topics/data/autobackup) feature. This feature is on by default and preserves app data, including **Shared Preferences**, which is what the **Preferences** API uses. You can disable this by following Google's [Auto Backup documentation](https://developer.android.com/guide/topics/data/autobackup).
 
 ## Limitations
 
 When storing a string, this API is intended to store small amounts of text. Performance may be subpar if you try to use it to store large amounts of text.
```

Comment:
Not convinced that subpar will localise well. Maybe something like "Performance may be reduced if..."?